2025-09-05 15:32:29,234 - large_scale_cognitive_test_client_11 - INFO - Starting client 11 with config: {'client_id': 11, 'server_address': '0.0.0.0:8080', 'experiment_name': 'large_scale_cognitive_test', 'seed': 467, 'batch_size': 32, 'epochs': 2, 'learning_rate': 0.001, 'attack_enabled': False}
2025-09-05 15:32:36,859 - large_scale_cognitive_test_client_11 - INFO - Client 11 initialized with 5955 training samples
2025-09-05 15:32:36,859 - large_scale_cognitive_test_client_11 - INFO - Client 11 connecting to 0.0.0.0:8080
2025-09-05 15:32:36,860 - flwr - WARNING - DEPRECATED FEATURE: flwr.client.start_client() is deprecated.
	Instead, use the `flower-supernode` CLI command to start a SuperNode as shown below:

		$ flower-supernode --insecure --superlink='<IP>:<PORT>'

	To view all available options, run:

		$ flower-supernode --help

	Using `start_client()` is deprecated.

            This is a deprecated feature. It will be removed
            entirely in future versions of Flower.
        
2025-09-05 15:32:36,873 - flwr - DEBUG - Opened insecure gRPC connection (no certificates were passed)
2025-09-05 15:32:36,874 - flwr - DEBUG - ChannelConnectivity.IDLE
2025-09-05 15:32:36,875 - flwr - DEBUG - ChannelConnectivity.TRANSIENT_FAILURE
2025-09-05 15:32:37,077 - flwr - DEBUG - gRPC channel closed
2025-09-05 15:32:37,078 - large_scale_cognitive_test_client_11 - ERROR - Client 11 failed: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:0.0.0.0:8080: Failed to connect to remote host: connect: Connection refused (61)"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:0.0.0.0:8080: Failed to connect to remote host: connect: Connection refused (61)", grpc_status:14}"
>
2025-09-05 15:38:46,389 - large_scale_cognitive_test_client_11 - INFO - Starting client 11 with config: {'client_id': 11, 'server_address': '0.0.0.0:8080', 'experiment_name': 'large_scale_cognitive_test', 'seed': 467, 'batch_size': 32, 'epochs': 2, 'learning_rate': 0.001, 'attack_enabled': False}
2025-09-05 15:38:54,064 - large_scale_cognitive_test_client_11 - INFO - Client 11 initialized with 5955 training samples
2025-09-05 15:38:54,064 - large_scale_cognitive_test_client_11 - INFO - Client 11 connecting to 0.0.0.0:8080
2025-09-05 15:38:54,064 - flwr - WARNING - DEPRECATED FEATURE: flwr.client.start_client() is deprecated.
	Instead, use the `flower-supernode` CLI command to start a SuperNode as shown below:

		$ flower-supernode --insecure --superlink='<IP>:<PORT>'

	To view all available options, run:

		$ flower-supernode --help

	Using `start_client()` is deprecated.

            This is a deprecated feature. It will be removed
            entirely in future versions of Flower.
        
2025-09-05 15:38:54,082 - flwr - DEBUG - Opened insecure gRPC connection (no certificates were passed)
2025-09-05 15:38:54,083 - flwr - DEBUG - ChannelConnectivity.IDLE
2025-09-05 15:38:54,084 - flwr - DEBUG - ChannelConnectivity.TRANSIENT_FAILURE
2025-09-05 15:38:54,291 - flwr - DEBUG - gRPC channel closed
2025-09-05 15:38:54,292 - large_scale_cognitive_test_client_11 - ERROR - Client 11 failed: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:0.0.0.0:8080: Failed to connect to remote host: connect: Connection refused (61)"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_status:14, grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:0.0.0.0:8080: Failed to connect to remote host: connect: Connection refused (61)"}"
>
2025-09-05 15:41:31,846 - large_scale_cognitive_test_client_11 - INFO - Starting client 11 with config: {'client_id': 11, 'server_address': '20.169.237.127:8080', 'experiment_name': 'large_scale_cognitive_test', 'seed': 467, 'batch_size': 32, 'epochs': 2, 'learning_rate': 0.001, 'attack_enabled': False}
2025-09-05 15:41:39,649 - large_scale_cognitive_test_client_11 - INFO - Client 11 initialized with 5955 training samples
2025-09-05 15:41:39,649 - large_scale_cognitive_test_client_11 - INFO - Client 11 connecting to 20.169.237.127:8080
2025-09-05 15:41:39,649 - flwr - WARNING - DEPRECATED FEATURE: flwr.client.start_client() is deprecated.
	Instead, use the `flower-supernode` CLI command to start a SuperNode as shown below:

		$ flower-supernode --insecure --superlink='<IP>:<PORT>'

	To view all available options, run:

		$ flower-supernode --help

	Using `start_client()` is deprecated.

            This is a deprecated feature. It will be removed
            entirely in future versions of Flower.
        
2025-09-05 15:41:39,666 - flwr - DEBUG - Opened insecure gRPC connection (no certificates were passed)
2025-09-05 15:41:39,667 - flwr - DEBUG - ChannelConnectivity.IDLE
2025-09-05 15:41:39,668 - flwr - DEBUG - ChannelConnectivity.CONNECTING
2025-09-05 15:41:39,909 - flwr - DEBUG - ChannelConnectivity.TRANSIENT_FAILURE
2025-09-05 15:41:40,112 - flwr - DEBUG - gRPC channel closed
2025-09-05 15:41:40,113 - large_scale_cognitive_test_client_11 - ERROR - Client 11 failed: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:20.169.237.127:8080: Failed to connect to remote host: connect: Connection refused (61)"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:20.169.237.127:8080: Failed to connect to remote host: connect: Connection refused (61)", grpc_status:14}"
>
